p4-16/spec/P4-16-spec.mdk

```diff
@@ -4083,8 +4134,12 @@ struct S {
     bit<32> b;
 }
 const S x = { 10, 20 };             // tuple expression
+                                    // implicit cast of tuple to struct
+                                    // implicit cast of int to bit<32>
 const S x = { a = 10, b = 20 };     // structure-valued expression
+                                    // implicit cast of int to bit<32>
```

Comment:
Again this is casting an anonymous struct to a struct but we cannot say it since we don't have anonymous struct type yet.